include/configs/zynq-common.h

```diff
@@ -268,17 +268,57 @@
 				"then env run importbootenv; " \
 			"fi; " \
 		"fi; \0" \
+	"refclk_source=internal\0" \
+	"mode=1r1t\0" \
 	"adi_loadvals=fdt addr ${fit_load_address} && fdt get value fdt_choosen /configurations/${fit_config}/ fdt && " \
 		"fdt get addr fdtaddr /images/${fdt_choosen} data && fdt addr ${fdtaddr}; "\
-		"if test -n ${ad936x_ext_refclk}  && test ! -n ${ad936x_skip_ext_refclk}; then " \
+		"if test -n ${ad936x_ext_refclk} && test ! -n ${ad936x_skip_ext_refclk}; then " \
 			"fdt set /clocks/clock@0 clock-frequency ${ad936x_ext_refclk}; " \
 		"fi; " \
-		"if test -n ${model}; then " \
-			"fdt set / model ${model}; " \
+		"fdt get value model / model; " \
+		"if test -n ${ad936x_ext_refclk_overwrite} && test \"${model}\" = \"Analog Devices PlutoSDR Rev.C (Z7010/AD9363)\"; then " \
```

Comment:
`ad936x_ext_refclk_overwrite` or `ad936x_ext_refclk_override`?

Reply:
i don't have a preference here. IF you think override is better i can change it.

Reply:
`override` sounds a bit more appropriate;